forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt

```diff
@@ -0,0 +1,9 @@
+Name:Trenzalore Clocktower
+ManaCost:no cost
+Types:Legendary Land
+A:AB$ Mana | Cost$ T | Produced$ R | SubAbility$ DBPutCounter | SpellDescription$ Add {U}.
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ TIME | SpellDescription$ Put a time counter on CARDNAME.
+A:AB$ ChangeZoneAll | Cost$ 1 U T SubCounter<12/TIME> Exile<1/CARDNAME> | IsPresent$ Time Lord.YouCtrl | Origin$ Graveyard,Hand | Destination$ Library | ChangeType$ Card.YouOwn | Shuffle$ True | SubAbility$ DBDraw | SpellDescription$ Shuffle your graveyard and hand into your library, then draw seven cards. Activate only if you control a Time Lord.
```

Comment:
do you mean you can't activate it?

Reply:
Yeah, I guess it has something to do with the Time Lord thing ?

Reply:
yeah the space for "Time Lord" in the Present Check might tip it off

@tool4ever you probably need to debug that?

Reply:
I think `CardType.parse()` does not work with it in some cases
but I don't know when/if I have time to fix it yet

Reply:
Try rebase and see if it works now

Reply:
Yes thanks 
![image](https://github.com/Card-Forge/forge/assets/67333662/ce068a17-9e4b-4c5f-b9ce-83b93601bc0c)